packages/react-docsite-components/src/components/Header/Header.tsx

```diff
@@ -36,7 +36,7 @@ export class HeaderBase extends React.Component<IHeaderProps, IHeaderState> {
     // For screen sizes large down, hide the side links.
     const sideLinks = isLargeDown ? [] : this.props.sideLinks;
 
-    const classNames = getClassNames(styles, { theme });
+    const classNames = getClassNames(styles, { theme, isLargeDown });
```

Comment:
This is a styling fix using some code which already exists in the header styles (for some reason `isLargeDown` wasn't being passed through previously)